code/src/library/Container.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const Container = styled.section`
+  display: grid;
+  grid-template-rows: min-content auto min-content;
```

Comment:
Since I set a fixed height on the Container, using min-content on all grid rows would drag them out, so setting auto on the middle row makes it take up all the space left after header and footer has taken up their min-content space